<launch>

  <include file="$(find hsr_description)/launch/upload_hsrb.launch"/>

  <node pkg="giskardpy" type="giskard.py" name="giskard" output="screen">
    <param name="config" value="HSR_StandAlone"/>
  </node>

  <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
    <rosparam param="enable_self_collision">False</rosparam>
    <rosparam param="interactive_marker_chains">
      - [odom, hsrb/base_footprint]
      - [odom, hsrb/head_rgbd_sensor_link]
      - [odom, hsrb/hand_palm_link]
    </rosparam>
  </node>

</launch>

